src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -417,6 +421,9 @@ async def _start_running(self) -> None:
         ]
         # now we wait for the server to actually start
         await self._server_started.wait()
+        # setup message queues
+        self._ensemble.scheduler_queue = self._events
```

Comment:
Should we rename `self._events` to something more precise?

Reply:
yes, we can :)

Reply:
Update: I've kept the `self._events` but made `Ensemble` stateless wrt. message queues. They are sent as attribute in `ensemble.evaluate` function.